deployments/roman/image/environments/post-start-hook

```diff
@@ -25,7 +25,8 @@ rsync -r  /etc/default-home-contents/ /home/jovyan
 
 # .............................................................................................
 
-git clone https://github.com/spacetelescope/romancal-notebooks.git
+# this will fail if the clone already exists
+git clone https://github.com/spacetelescope/romancal-notebooks.git || true
```

Comment:
Is there a reason not to use git-sync for this?  git clone was an oversight.

Reply:
Ah, OK.  I can certainly do that.